broker-cli/proto/broker.proto

```diff
@@ -309,8 +309,11 @@ service AdminService {
  */
 enum TimeInForce {
   GTC = 0; // Good-til-cancelled, an order that remains in force until cancelled by the user
+  // FOK is not supported
```

Comment:
maybe best to comment out the actual definition of `FOK`

Reply:
@dannypaz Would you be okay with commenting it out?

@treygriffith, Danny and I had an in-person chat about commenting out the definitions in the `enum` and decided to handle the unsupported definition within the implementation rather than having commented out code.

Reply:
It was a tomato/tomato thing. Basically, we've had these uncommented + throw an exception for unsupported, so we just decided to keep them in